server/documents/elements/icon.html.eco

```diff
@@ -193,10 +209,8 @@ themes      : ['Default']
         <div class="column"><i class="share icon"></i>share</div>
         <div class="column"><i class="share square icon"></i>share square</div>
         <div class="column"><i class="share square outline icon"></i>share square outline</div>
-        <div class="column"><i class="sign-in icon"></i>sign-in</div>
```

Comment:
`sign in` not alternate version does not exits anymore ?

Reply:
They are a SUI thing, something weird happened here https://github.com/Semantic-Org/Semantic-UI/commit/363f283915619170366e41ad6dad70e1fce5fee2#diff-9760f14199449f64f501a20559a20dad